tools/eksDistroBuildToolingOpsTools/pkg/eksGoRelease/eksGoRelease.go

```diff
@@ -0,0 +1,409 @@
+func (r Release) NewMinorRelease(ctx context.Context) error {
+	r.Release = 0
+	// Setup Github Client
+	retrier := retrier.New(time.Second*380, retrier.WithBackoffFactor(1.5), retrier.WithMaxRetries(15, time.Second*30))
+
+	token, err := github.GetGithubToken()
+	if err != nil {
+		logger.V(4).Error(err, "no github token found")
+		return fmt.Errorf("getting Github PAT from environment at variable %s: %v", github.PersonalAccessTokenEnvVar, err)
+	}
+
+	githubClient, err := github.NewClient(ctx, token)
+	if err != nil {
+		return fmt.Errorf("setting up Github client: %v", err)
+	}
+
+	// Creating git client in memory and clone 'eks-distro-build-tooling
+	forkUrl := fmt.Sprintf(githubRepoUrl, sOwner)
+	gClient := git.NewClient(git.WithInMemoryFilesystem(), git.WithRepositoryUrl(forkUrl), git.WithAuth(&http.BasicAuth{Username: sOwner, Password: token}))
+	if err := gClient.Clone(ctx); err != nil {
+		logger.Error(err, "Cloning repo")
+		return err
+	}
+	// Create new branch
+	if err := gClient.Branch(r.EksGoReleaseFullVersion()); err != nil {
+		logger.Error(err, "git branch", "branch name", r.EksGoReleaseFullVersion(), "repo", forkUrl, "client", gClient)
+		return err
+	}
+
+	// Add files for new minor versions of golang.
+	// Add README.md
+	readmePath := fmt.Sprintf(basePathFmt, projectPath, r.GoMinorReleaseVersion(), readme)
+	readmeContent := []byte(generateReadme(r.GoMinorReleaseVersion(), r.GoFullVersion(), r.GoSemver(), r.ReleaseNumber()))
+	if err := gClient.CreateFile(readmePath, readmeContent); err != nil {
+		logger.Error(err, "Adding README.md", "path", readmePath, "content", readmeContent)
+		return err
+	}
+	if err := gClient.Add(readmePath); err != nil {
+		logger.Error(err, "git add", "file", readmePath)
+		return err
+	}
+
+	// Add RELEASE
+	releasePath := fmt.Sprintf(basePathFmt, projectPath, r.GoMinorReleaseVersion(), ghRelease)
+	releaseContent := []byte(fmt.Sprintf("%d", r.ReleaseNumber()))
+	if err := gClient.CreateFile(releasePath, releaseContent); err != nil {
+		logger.Error(err, "Adding RELEASE", "path", releasePath, "content", releaseContent)
+		return err
+	}
+	if err := gClient.Add(releasePath); err != nil {
+		logger.Error(err, "git add", "file", releasePath)
+		return err
+	}
+
+	// Add GIT_TAG
+	gittagPath := fmt.Sprintf(basePathFmt, projectPath, r.GoMinorReleaseVersion(), gitTag)
+	gittagContent := []byte(fmt.Sprintf("go%s", r.GoFullVersion()))
+	if err := gClient.CreateFile(gittagPath, gittagContent); err != nil {
+		logger.Error(err, "Adding GIT_TAG", "path", gittagPath, "content", gittagContent)
+		return err
+	}
+	if err := gClient.Add(gittagPath); err != nil {
+		logger.Error(err, "git add", "file", gittagPath)
+		return err
+	}
+
+	// Add fedora.go
+	fedoraFilePath := fmt.Sprintf(rpmSourcePathFmt, projectPath, r.GoMinorReleaseVersion(), fedora)
+	fedoraContent := []byte("temp")
```

Comment:
might be nice to have the content of these be links to the other implementations or some kind of other useful comment/documentation. That way someone can run this, generate the scaffolding, and then just follow on from there

Reply:
Even just explain how to get these files. Agreed that is better than "temp"